src/fg_apply.rs

```diff
@@ -133,7 +133,7 @@ pub(crate) fn rav1d_apply_grain_row<BD: BitDepth>(
     let seq_hdr = &***out.seq_hdr.as_ref().unwrap();
     let frame_hdr = &***out.frame_hdr.as_ref().unwrap();
     let data = &frame_hdr.film_grain.data;
-    let data_c = &data.clone().into();
+    let data_c = &data.clone();
```

Comment:
These (`data` and `data_c`) were previously different `Rav1d*` and `Dav1d*` types, but they've now been unified and we must've forgotten to update this.  I think we can elide the `.clone()` here, too, and just have `let data_c = data`.

```suggestion
    let data_c = data;
```